README.md

```diff
@@ -89,7 +89,7 @@ This command will:
    This tool is optional and if you leave it out you can always attach it to an already running
    `geth` instance with `geth attach`.
 
-### A Full node on the Görli test network
+### A Full node on the Göerli test network
```

Comment:
Goerli has been deprecated for a while, could you please switch the testnet here to holesky?